wallet.py

```diff
@@ -303,18 +326,18 @@ def getUTXOs(self, requested, approve=None):
         return matches, collected >= requested
     def getKey(self, addr):
         """
-        Get the PrivateKey for the provided address.
+        Get the private key for the provided address.
 
-        Args: 
+        Args:
             addr (str): The base-58 encoded address.
 
         Returns:
-            PrivateKey: The private key structure for the address.
+            privateKey (str): The private key structure for the address.
```

Comment:
Perhaps this is an object?

Reply:
It does return an object, which is an instance of a `secp256k1.PrivateKey`.